pkg/tanka/evaluators.go

```diff
@@ -63,7 +63,7 @@ local noDataEnv(object) =
          && std.objectHas(object, 'kind')
       then
         if object.kind == 'Environment'
-        then object { data:: {} }
+        then object { data:: super.data }
```

Comment:
```suggestion
        then object { data+:: {} }
```

My natural choice would have been above. Is there any functional difference?

Reply:
I don't think so.